vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -375,6 +375,12 @@ - (void)activityInfoWithFilter:(nullable BATActivityInfoFilter *)filter completi
   });
 }
 
+- (void)publisherActivityFromURL:(NSURL *)URL
+{
+    auto visitData = [self visitDataForURL:URL tabId:0];
+    ledger->GetPublisherActivityFromUrl(1, visitData.Clone(), std::string());
```

Comment:
Why not just adjust the overall method and make `faviconURL` and `publisherBlob` nullable and then make this method call that one.